Kernel_23/test/Kernel_23/Simple_cartesian.cpp

```diff
@@ -80,9 +77,21 @@ main()
   _test_cls_iso_cuboid_3( Cls() );
   _test_angle( Cls() );
 
-  std::cout << "Testing 3d-2d with Simple_cartesian<Quotient<Precise_integer>>:";
+  std::cout << "Testing 3d-2d with "+kernel_name+" :";
   std::cout << std::endl;
   _test_mf_plane_3_to_2d( Cls() );
+}
 
+int
+main()
+{
+  typedef   CGAL::Simple_cartesian<double>     Clsd;
+  std::cout << "Testing IO with Simple_cartesian<double> :" << std::endl;
+  _test_io( Clsd() );
+
+  typedef   CGAL::Simple_cartesian<CGAL::Quotient<Precise_integer> >     Cls;
+  test_kernel("Simple_cartesian<Quotient<Precise_integer>>", Cls());
+  typedef   CGAL::Simple_cartesian<CGAL::Quotient<Precise_integer> >     Cls;
+  test_kernel("Epick_without_intervals", CGAL::Epick_without_intervals());
```

Comment:
fails.

Reply:
Indeed the tests are completely wrong. We have nothing in CGAL to test correctly `CGAL::Epick`!

Reply:
I have eventually managed to set up a testsuite for `CGAL::Epick`. And then also for `CGAL::Epick_without_intervals`, in https://github.com/CGAL/cgal/pull/3939/commits/7d543d5bf80ae5fe3d0075d52b9750e3a0115fbc.